src/course-unit/course-sequence/sequence-navigation/SequenceNavigation.jsx

```diff
@@ -0,0 +1,112 @@
+import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import {
+  injectIntl, intlShape, isRtl, getLocale,
+} from '@edx/frontend-platform/i18n';
+import { Button, useWindowSize, breakpoints } from '@edx/paragon';
+import {
+  ChevronLeft as ChevronLeftIcon,
+  ChevronRight as ChevronRightIcon,
+} from '@edx/paragon/icons';
+
+import { useModel } from '../../../generic/model-store';
+import { RequestStatus } from '../../../data/constants';
+import { getSequenceStatus } from '../../data/selectors';
+import { useSequenceNavigationMetadata } from '../hooks';
+import messages from '../messages';
+import SequenceNavigationTabs from './SequenceNavigationTabs';
+
+const SequenceNavigation = ({
+  intl,
+  unitId,
+  sequenceId,
+  className,
+}) => {
+  const sequenceStatus = useSelector(getSequenceStatus);
+  const {
+    isFirstUnit, isLastUnit, nextLink, previousLink,
+  } = useSequenceNavigationMetadata(sequenceId, unitId);
+  const sequence = useModel('sequences', sequenceId);
+
+  const shouldDisplayNotificationTriggerInSequence = useWindowSize().width < breakpoints.small.minWidth;
+  const renderUnitButtons = () => {
+    if (sequence.unitIds?.length === 0 || unitId === null) {
+      return (
+        <div style={{ flexBasis: '100%', minWidth: 0, borderBottom: 'solid 1px #EAEAEA' }} />
+      );
+    }
+
+    return (
+      <SequenceNavigationTabs
+        unitIds={sequence.unitIds || []}
+        unitId={unitId}
+      />
+    );
+  };
+
+  const renderPreviousButton = () => {
```

Comment:
If there is no previous unit or subsection hide previous button instead of disabling

Reply:
Corrected